otx/algorithms/classification/adapters/mmcls/datasets/pipelines/transforms/augments.py

```diff
@@ -1,6 +1,9 @@
+"""Module for defining Augments and CythonArguments class used for classification task."""
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
+# pylint: disable=unused-argument, no-value-for-parameter
+# mypy: ignore-errors
```

Comment:
Should you need to disable mypy for the whole file? Please set it as much as necessary